docs/kpt_functions.md

```diff
@@ -0,0 +1,59 @@
+# Policy Library KPT Functions
+This repo contains several kpt functions to manage documentation and make it easier to pull in policy for Config Validator.
+
+These functions are meant to be run against this repo. Before running one of the below functions run `kpt get`:
+
+```
+kpt pkg get https://github.com/forseti-security/policy-library.git ./policy-library
+cd ./policy-library
+```
+
+## Generate Docs
+Generates markdown documentation for the templates and constraints in this repo.
+
+### Run with Node
+Generate the markdown docs.
+
+```
+kpt fn source ./samples/ ./policies/ | 
+node bundler/dist/generate_docs_run.js -d sink_dir=./docs -d overwrite=true
+```
+
+## Get Policy Bundle
+Filters the samples in this repo for a given [policy bundle](./index.md).
+
+### Run with Node
+Copy the `forseti-security` bundle to the constraints directory using Node.
+
+```
+kpt fn source ./samples/ |
+node bundler/dist/get_policy_bundle_run.js -d bundle=bundles.validator.forsetisecurity.org/forseti-security -d sink_dir=./policies/constraints/ -d overwrite=true
+```
+
+### Build Docker image
```

Comment:
Could we put these in a separate dev guide?

Reply:
Will do